en/advanced/gimbal_control.md

```diff
@@ -1,20 +1,37 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
+If you want to control a gimbal with a camera (or any other payload) attached to
+the vehicle, you need to configure how you want to control it and how PX4 can
```

Comment:
Ok, so I didn't make it clear enough: this will be reformated when published, as specified by the markdown language, see https://toolchain.gitbook.com/syntax/markdown.html, section **Paragraphs and Line Breaks**.